tsconfig.json

```diff
@@ -38,7 +38,9 @@
     // error out if import and file system have a casing mismatch. Recommended by TS
     "forceConsistentCasingInFileNames": true,
     // `tsdx build` ignores this option, but it is commonly used when type-checking separately with `tsc`
-    "noEmit": true,
     "jsx": "react",
+    "declarationDir": "dist/types",
+    "outDir": "dist",
+    "emitDeclarationOnly": true
```

Comment:
can we add 

`"noEmit": true,`

I believe its used to enable type checking at runtime, but could be wrong 😕